Documents/API.md

```diff
@@ -262,6 +262,17 @@ Example response, with HTTP code 400:
 
 `GET` - Returns all parsing errors for a submission
 
+This endpoint is paginated.
+If no query parameter is included, it will return the first 20 parse errors for the file.
+Include the `page` param to access any other pages. Ex: `.../parseErrors?page=3`
```

Comment:
What do you think about adding a table here for the query parameters? Just to be consistent?
[Example](https://github.com/schbetsy/hmda-platform/blob/a550d1f67619aa6025ac18a3c86b7fb4f2ea55d9/Documents/API.md#edits)